av1an-cli/src/lib.rs

```diff
@@ -514,10 +509,19 @@ pub fn parse_cli(args: CliOpts) -> anyhow::Result<EncodeArgs> {
     chunk_order: args.chunk_order,
     concat: args.concat,
     encoder: args.encoder,
-    extra_splits_len: if args.extra_split > 0 {
-      Some(args.extra_split)
-    } else {
-      None
+    extra_splits_len: match args.extra_split {
+      Some(x) => {
+        if x > 0 {
+          Some(x)
+        } else {
+          None
+        }
+      }
+      // Make sure it's at least 10 seconds
+      None => match input.frame_rate() {
+        Ok(fps) => Some((fps * 10.0) as usize),
```

Comment:
We probably want to cap the maximum to some reasonable value so as to not choose unreasonably high values for high framerate content.

Reply:
10 seconds seems reasonable to tome